Compile-Betaflight.ps1

```diff
@@ -0,0 +1,170 @@
+<#
+Use this to create .hex files from Betaflight Code.

```

Comment:
You should probably add that this will only ever work on Windows because your paths are Windows specific.

Also, please note that this way of building Betaflight on Windows has been superseded by using WSL, which provides a much better experience. It is only kept around for users of versions of Windows (or non-Windows-OS) that do not support WSL.